tests/integration/helpers.py

```diff
@@ -5,11 +5,47 @@
 import logging
 from typing import Optional
 
+from lightkube import AsyncClient
+from lightkube.resources.core_v1 import Service
 from pytest_operator.plugin import OpsTest
 
 logger = logging.getLogger(__name__)
 
 
+async def get_k8s_service_address(ops_test: OpsTest, service_name: str) -> Optional[str]:
```

Comment:
Specifically for traefik, because we already know it must be loadBalancer, wdyt about replacing this func with a one-liner such as:
```python
sh.kubectl(*"-n test-bundle-iwfn get service/trfk -o=jsonpath='{.status.loadBalancer.ingress[0].ip}'".split())
```